frontend/src/app/main/ui/dashboard/fonts.cljs

```diff
@@ -103,6 +103,10 @@
                          (fn [error]
                            (js/console.log "error" error))))))
 
+        on-upload-all
+        (fn [items]
+          (dorun (map on-upload items)))
```

Comment:
For this purpose is much better use just `run!` (replace the `dorun + map` with `run!`)

Reply:
Thank you very much!, I had never used run! before :)